apidoc/v2/openapi.yaml

```diff
@@ -1364,10 +1364,26 @@ paths:
                         'exclusiveeditmode',
                         'lock',
                         'pin',
+                        'sign',
                         'timestamp',
                         'updatemetadatafield',
                         'update',
                       ]
+                      default: 'sign'
+                    passphrase:
+                      type: string
+                      default: 'your-passphrase'
```

Comment:
there is no default value for this param.

Reply:
yes my bad, i assigned it this way so that the example doesn't appear this way
![image](https://github.com/user-attachments/assets/1262d296-d33a-4fb0-874e-a87766d0b41b)

But i guess the description you suggested will do the job well ! thanks